app/components/Wallet/useWallet.tsx

```diff
@@ -93,5 +118,100 @@ export const useWallet = () => {
 		if (response.status === "success") setNetwork(newNetwork);
 	}, []);
 
-	return { isConnected, balance, network, connectWallet, disconnectWallet, switchNetwork };
+	const fetchUTXOs = useCallback(async (address: string): Promise<UtxoI[]> => {
+		try {
+			const response = await axios.get(`${MEMPOOL_API}/address/${address}/utxo`);
+			return response.data.map((utxo: UtxoI) => ({
+				txid: utxo.txid,
+				vout: utxo.vout,
+				value: utxo.value,
+				scriptPubKey: utxo.scriptpubkey,
+			}));
+		} catch (error) {
+			throw new Error(`Failed to fetch UTXOs: ${error}`);
+		}
+	}, []);
+
+	const fetchValidateAddress = useCallback(async (address: string): Promise<ValidateAddressI> => {
+		try {
+			const response = await axios.get(`${MEMPOOL_API}/v1/validate-address/${address}`);
+			return response.data;
+		} catch (error) {
+			throw new Error(`Failed to fetch UTXOs: ${error}`);
+		}
+	}, []);
+
+	const sendTxn = useCallback(async () => {
+		// bitcoin wallet address
+		const bitcoinAddress = addressInfo.find((adr) => adr.purpose === AddressPurpose.Payment);
+		if (!bitcoinAddress) return;
+		// fetch utxos
+		const utxos: UtxoI[] = await fetchUTXOs(bitcoinAddress.address);
+		if (!utxos?.length) return;
+		// validate address
+		const validateAddress: ValidateAddressI = await fetchValidateAddress(bitcoinAddress.address);
+		if (!validateAddress) return;
+
+		const network = bitcoin.networks.testnet;
+		const psbt = new bitcoin.Psbt({ network });
+
+		psbt.addInput({
+			hash: utxos?.[0]?.txid,
+			index: utxos?.[0]?.vout,
+			witnessUtxo: {
+				script: Buffer.from(validateAddress.scriptPubKey, "hex"),
+				value: utxos?.[0]?.value,
+			},
+		});
+
+		const sendAmount = 1000;
+		psbt.addOutput({
+			// TODO: replace hardcoded P2WPKH address for nBTC deposits
+			address: "tb1qe60n447jylrxa96y6pfgy8pq6x9zafu09ky7cq",
+			value: sendAmount,
+		});
+
+		// Add OP_RETURN output
+		// TODO: hardcoded sui address
+		const opReturnData = Buffer.from(
+			"0x0dfeef16c6730d27c1b53ba3b96c75831c2fbc66882b3ff136513bbdce9c60ea",
```

Comment:
what is this address?

Reply:
this is sui address that has been stored as OP RETURN data.

Reply:
OK, please add it somewhere to the config.